### ISSUE TITLE
fix treesitter examples in wiki

### ISSUE THREAD

**Issue by matu3ba** [marko-cerovac/material.nvim#52]
They are broken as you can see [here](https://github.com/nvim-treesitter/nvim-treesitter/wiki/Colorschemes#marko-cerovacmaterialnvim).

Quite a shame, since your plugin provides proper highlight groups for all colorschemes and only contestors are navarasu/onedark.nvim (no light color) and adisen99/codeschool.nvim (key words for control flow have bad contrast and color ).

You could maybe elaborate in 2-3 sentences on what color groups one wants to choose for good programming experience.

**Comment by marko-cerovac** [comment]
Thanks for noticing, I've fixed the bad links.
I wrote the material.nvim plugin, but I didn't invent the colors. The colors have been chosen by people with much more knowledge and good taste for colors than me. I followed the [material theme palette](https://www.material-theme.com/docs/reference/color-palette/) and tried to stay as close to the original as possible. I think it mostly comes down to the balance of contrast. This link contains color palettes for other well known themes like Moonlight, Dracula, One-Dark, Solarized etc.

**Comment by marko-cerovac** [comment]
Thanks for noticing, I've fixed the bad links.
I wrote the material.nvim plugin, but I didn't invent the colors. The colors have been chosen by people with much more knowledge and good taste for colors than me. I followed the [material theme palette](https://www.material-theme.com/docs/reference/color-palette/) and tried to stay as close to the original as possible. I think it mostly comes down to the balance of contrast. This link contains color palettes for other well known themes like Moonlight, Dracula, One-Dark, Solarized etc.
